client/state/plugins/wporg/reducer.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import React from 'react/addons';
+
+function updatePluginState( state = {}, pluginSlug, attributes ) {
+	return Object.assign( {},
+		state,
+		{ [ pluginSlug ]: Object.assign( {}, state[ pluginSlug ], attributes ) }
+	);
+}
```

Comment:
yup, I've seen used in other reducers... but in this case, since this plugins data is always just one level deep, I didn't think it were necessary to use it. And since the sintax is lightly more complex that just to use a `Object.assign`, I favored the simpler option. Is there any advantage of using `React.addons.update` apart from tidy in-deep merges?